templates/web/pages/rsvp.tmpl

```diff
@@ -36,30 +36,40 @@
 					</div>
 
           <div id="rsvp-yes-selected">
-            <p class=" has-text-centered subtitle">Get ready to hack your own adventure.</p>
+            <p class=" has-text-centered subtitle" style="font-weight: 500">Get ready to lay back and hack.</p>
             <p>You'll receive an email with more logistical information in the coming weeks.</p>
 
             <div class="field is-horizontal">
               <div class="field-label is-normal">
-                <label for="accommodations" class="label">Required accommodations</label>
+                <label for="accommodations" class="label">Shipping Address (for swag)</label>
               </div>
               <div class="field-body">
                 <div class="field">
                   <div class="control">
-                    <input type="text" name="accommodations" class="input" maxlength="1024" value="{{ .FormRefill.Accommodations}}">
+                    <input type="text" name="shipping-address" class="input" maxlength="1024" value="{{ .FormRefill.ShippingAddr}}">
```

Comment:
This seems like an awkward way to collect shipping info, usually it's broken up into sections for readability and better UX

Reply:
wdym? like 1) street address 2) city 3) country 4) zip code?

Reply:
yeah